### PR TITLE
Add a custom log panel

### LINKED ISSUE #99: log panel with override config
### What would you like to be added?

Create a grafana log panel, which can config field as data link. 

### Why is this change required? What problem does it solve?

Through this configuration, the field of the log can support datalinks

### PR DESCRIPTION
**Pull request by LLKCoder**
##Description
Create a grafana log panel with override function.
At label area, users can customize the display content and format of the field.

##Usage
1.Enable the panel plugin
截屏2024-03-14 11 40 14

2.Add a custom log panel to dashboard

3.config your own field override

截屏2024-03-14 11 44 06

##RelatedIssues
Close #99

### CODE REVIEW

File: build/docker/Dockerfile.grafana [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
 RUN npm install \
     && npm run build
-
 # Use the official Grafana image as the base image.
 FROM grafana/grafana:latest
 
-COPY --from=node /app/dist /var/lib/grafana/plugins/antgroup-yaml-panel
-
-# Define the list of plugins to install, separated by spaces.
-ARG PLUGINS=""
```
[the remark this answers is not in the document]
Comment (LLKCoder) [reply build/docker/Dockerfile.grafana:17]: ok

**Comment by hnhbwlp** [comment]
Can you show the demo of this feature with image?

**Review by d3c3mber — APPROVED** [review]
lgtm

**Comment by LLKCoder** [comment]
@hnhbwlp 
截屏2024-03-18 14 04 00

**Review by hnhbwlp — APPROVED** [review]

**Pull request merged**